src/ethereum_spec_tools/evm_tools/utils.py

```diff
@@ -12,6 +14,39 @@
 
 W = TypeVar("W", Uint, U64, U256)
 
+exception_maps = {
```

Comment:
Probably should have an upper case name if it's a constant:

```suggestion
EXCEPTION_MAPS = {
```

Reply:
updated